src/components/Page.tsx

```diff
@@ -5,22 +5,27 @@ import { pageContext } from '../utils/pageContext';
 import { ExternalPageProps } from '../utils/ssrProps';
 
 import { Header } from './Header';
+import { Footer } from './Footer';
+import styled from 'styled-components';
```

Comment:
Looks like incorrect imports order. Do you some errors from ESLint in your editor?

Reply:
done